src/into_dart.rs

```diff
@@ -296,6 +303,35 @@ where
 
 impl<T> IntoDartExceptPrimitive for Vec<T> where T: IntoDartExceptPrimitive {}
 
+impl<T> IntoDart for HashSet<T>
+where
+    T: IntoDartExceptPrimitive,
+{
+    fn into_dart(self) -> DartCObject {
+        self.into_iter().collect::<Vec<_>>().into_dart()
+    }
+}
```

Comment:
Looks reasonable, we should mention that even tho it returns a Vec<T> as a transport format, this does not mean the items in the result Vector are the same order in the HashSet.

Reply:
IMHO HashSet are unordered:

![image](https://github.com/shekohex/allo-isolate/assets/5236035/0d8143dd-b62c-4503-a416-a7c0404827e0)

Reply:
Exactly, thus I wanted to make it clear that the expected behavior here is that the items in the result Vec are not in order, even tho it is a List<T>.